definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-general-referral-nonprod.json

```diff
@@ -0,0 +1,296 @@
+[
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "GeneralReferralReason",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
```

Comment:
please use CRUD

It's going to be "re-triggerable" :)

I don't know why, but when I didn't have D it didn't work.

Reply:
10-4, would update them all